exampleCourse/questions/element/checkbox/question.html

```diff
@@ -257,3 +257,28 @@
         </pl-checkbox>
     </div>
 </div>
+
+<div class="card my-2">
+    <div class="card-header">
+        Part 10
+    </div>
+
+    <div class="card-body">
+        <pl-question-panel>
+            <p>Here, we set <code>min-select="3"</code> and <code>max-select="8"</code> to restrict the number of answers that can be selected.
+                Any submission that selects less than 3 answers or more than 8 answers is marked as invalid.</p>
+        </pl-question-panel>
+
+        <pl-checkbox answers-name="group_ans_v9" hide-letter-keys="true" min-select="3" max-select="8">
+            <pl-answer correct="true">Exposure to new viewpoints from different fields and life experiences.</pl-answer>
+            <pl-answer correct="true">Improved creativity and overall work quality.</pl-answer>
+            <pl-answer>               The ability to work on a project by yourself.</pl-answer>
+            <pl-answer>               Having only one person make all of the decisions.</pl-answer>
+            <pl-answer correct="true">Constructive dialog and increased internal group motivation.</pl-answer>
+            <pl-answer correct="true">Personal accountability to the team to work on a project.</pl-answer>
+            <pl-answer correct="true">Friendship.</pl-answer>
+            <pl-answer>               The resources to work on a small project instead of a large project. </pl-answer>
+            <pl-answer correct="true">Prepares you for a team-environment in the workplace or a research group.</pl-answer>
+        </pl-checkbox>
+    </div>
+</div>
```

Comment:
@Cris-Moran did you add this to the wrong PR?

Reply:
I don't remember writing this. I believe this appeared when I pulled from upstream.

Reply:
Doesn't matter, I removed it, it was causing tests to fail and is not relevant anyway.